packages/postcss-ordered-values/src/rules/columns.js

```diff
@@ -0,0 +1,27 @@
+import { unit } from 'postcss-value-parser';
+
+const strValues = ['auto', 'inherit', 'unset', 'initial'];
+
+export default (columns) => {
+  let newValue = { front: '', back: '' };
+  let shouldNormalize = false;
+  columns.walk((node) => {
+    const { type, value } = node;
+    if (type === 'word' && strValues.indexOf(value.toLowerCase())) {
```

Comment:
```suggestion
    if (type === 'word' && strValues.includes(value.toLowerCase())) {
```

Reply:
I need to revert it, its failing the tests

Reply:
Oops! I did found couple of instances where `Array.includes()` was used for the purpose.

Reply:
👍

Reply:
https://github.com/cssnano/cssnano/search?q=includes&unscoped_q=includes

Reply:
Not sure why then